test-integration/test_integration/test_predict.py

```diff
@@ -28,6 +30,7 @@ def test_predict_takes_int_inputs_and_returns_ints_to_stdout():
     )
     # stdout should be clean without any log messages so it can be piped to other commands
     assert result.stdout == b"4\n"
+    assert "falling back to slow loader" not in str(result.stderr)
```

Comment:
Hmmm, this feels like a very brittle way to test that we're using the "fast loader". I'm also not really sure about us emitting these logs into the setup output that aren't going to mean anything to the people that wrote the models.

I don't think this is blocking but is there perhaps a separate set of standalone tests that look a bit less like integration tests that are reliant on log output?

I was thinking something like:

```python
@pytest.mark.parametrize("fixture", ALL_FIXTURES)
def test_slim_loader_generates_correct_model(fixture):
    model_slim = load_slim(fixture)
    model_full = load_full(fixture)

    assert model_slim.model_dump() == model_full.model_dump()
```

Reply:
@nickstenning 
uploaded 
- disable emitting fast loader logs into the setup 
- added test that compares fast vs slow predict signatures. it goes over all fixtures in server tests.